packages/language-server/src/lib/MessageHandler.ts

```diff
@@ -192,54 +202,49 @@ export function handleHoverRequest(document: TextDocument, params: HoverParams):
     return
   }
 
-  const blockDocumentation = getDocumentationForBlock(document, block)
+  const blockDocumentation = getDocumentationForBlock(block)
 
   if (blockDocumentation.length !== 0) {
     return {
       contents: blockDocumentation.join('\n\n'),
     }
   }
-
-  // TODO uncomment once https://github.com/prisma/prisma/issues/2546 is resolved!
-  /*if (docComments.startsWith('//')) {
-    return {
-      contents: docComments.slice(3).trim(),
-    }
-  } */
-
-  return
```

Comment:
Is this intentionally removed rather than uncommented?

Reply:
Yes, I don't think what written here makes sense anymore.